.go-version

```diff
@@ -1 +1 @@
-1.70.12
+1.70.13
```

Comment:
Is this intentional?

Reply:
This seems correct. We should be bumping the version on every change.

Reply:
Yeah, iirc this isn't used anywhere for official releases but is used for local builds during development

Reply:
FYI, this is used for building the official images: https://github.com/lightstep/terraform-provider-lightstep/blob/main/Makefile#L5